kpi/views/v1/export_task.py

```diff
@@ -86,3 +86,7 @@ def create(self, request, *args, **kwargs):
                 request=request),
             'status': ExportTask.PROCESSING
         }, status.HTTP_201_CREATED)
+
+    def perform_destroy(self, instance):
+        ExportTask.result.field.storage.delete(name=str(instance.result))
+        return super().perform_destroy(instance)
```

Comment:
Hey @joshuaberetta,
* Would `instance.result.delete(save=False)` work? [[ref](https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.fields.files.FieldFile.delete)]
* Could this go into a `delete()` method on the `ExportTask` model? [Some deletions may happen through the ORM directly](https://github.com/kobotoolbox/kpi/blob/bf0c8cb6d6804c2bd2d4e78d1f768cac2c121abb/kpi/models/import_export_task.py#L648-L651), not through the HTTP API, and it'd be nice to do the same clean up in those cases.

Reply:
Thanks for the suggestion! I'll look into that 💪